sound/soc/sof/intel/hda-dai.c

```diff
@@ -414,6 +414,49 @@ static struct snd_soc_cdai_ops sof_probe_compr_ops = {
 #endif
 #endif
 
+static int ssp_dai_hw_params(struct snd_pcm_substream *substream,
```

Comment:
prefix needs to be fixed to ASoC: SOF: Intel: HDA

Reply:
ok